packages/playwright/src/takeSnapshot.ts

```diff
@@ -6,7 +6,8 @@ import { logger } from '@chromatic-com/shared-e2e';
 
 const rrweb = readFileSync(require.resolve('rrweb-snapshot/dist/rrweb-snapshot.js'), 'utf8');
 
-export const contentType = 'application/rrweb.snapshot+json';
+// top-level key is the test ID, next level key is the name of the snapshot (which we expect to be unique)
```

Comment:
This snapshot-names-should-be-unique-wothin-the-same-test assumption is also currently used by us -- if two manual snapshots (within the same test) use the same title, we'll only end up with one story. So I didn't try to change that for now.